mobile/src/containers/GameSessionContainer.js

```diff
@@ -6,6 +6,8 @@ async function storeGameSessionLocal(gameSession, teamId, teamMember, gameCode)
     try {
         const localGameSession = await EncryptedStorage.setItem(
             "localGameSession",
+            // TODO: maybe we should spread the existing value here and only add 
+            // values with updates
```

Comment:
In general, we should only store the data needed, `gameCode`, `teamId` and `memberId` and refetch the whole game session after loading from storage and set the team and members and go to the right screen